src/plugins/navigation/next.js

```diff
@@ -232,6 +232,9 @@ export default pluginFactory({
             .on('loaditem', () => {
                 updateElement(this.$element, isLastItem());
             })
+            .on('renderitem', () => {
+                $("h2#test-title-header").attr('tabindex', -1).attr('role', 'alert').focus();
```

Comment:
Did you check this with enabled plugin 'taoQtiTest/runner/plugins/content/accessibility/focusOnFirstField' ?

Reply:
deprecated after corrected acceptance criteria